recipes/pyjnius/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: pyjnius
+
+  {% set tagged_version = GIT_DESCRIBE_TAG %}
+  # If current revision is not tagged add number of commits since last tag to version.
+  {% if GIT_DESCRIBE_NUMBER|int != 0 %}
+    {% set tagged_version = tagged_version + '.post' + GIT_DESCRIBE_NUMBER %}
+  {% endif %}
+
+  version: {{tagged_version}}
+
+source:
+  git_url: https://github.com/kivy/pyjnius.git
+  git_rev: "61149f20436e560cdff89e4abdd9035f8e4f7d35"
+
+build:
+  number: 0
+  string: py{{py}}_{{PKG_BUILDNUM}}
+
+requirements:
+ build:
+   - {{ compiler('c') }}
+   - {{ compiler('cxx') }}
+ host:
+   - openjdk >=8
+   - python {{PY_VER}}*
+   - cython
```

Comment:
you'll want to add `pip` here

Reply:
Think this is causing the build failure on Linux